File: LongChallenge_2018_10_CHSERVE.cpp
// code_report Solution
// https://youtu.be/GMY0D60TPWE

int main() {
   int t; cin >> t;
   while (t--) {
      int a, b, k;
      cin >> a >> b >> k;
      cout << ((a + b) % (2 * k) < k ? "CHEF\n" : "COOK\n");
   }
   return 0;
}

File: LongChallenge_2018_10_CHSERVE.py
# code_report Solution
# https://youtu.be/GMY0D60TPWE

t = int(input())
for _ in range(t):
    a, b, k = [int(i) for i in input().split()]
    print('CHEF') if (a + b) % (2 * k) < k else print('COOK')
